docs/concepts/advanced/provide_inject.md

```diff
@@ -82,7 +82,6 @@ class ChildComponent(Component):
         my_data = self.inject("my_data")
         print(my_data.hello)    # hi
         print(my_data.another)  # 123
-        return {}
```

Comment:
Unrelated to the URL managing script, but there were also a lot of instances of `get_template_data()` where we return empty dict. That's no longer required, so I removed them.